native/packages/react-native-bpk-component-banner-alert/readme.md

```diff
@@ -23,10 +23,10 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
     justifyContent: 'center',
-    padding: spacingBase,
+    padding: TOKENS.spacingBase,
```

Comment:
This might just be a consequence of the rebase, but I don't think this is right!

Drop the `TOKENS`, just `spacingBase`. It's cleaner.